06-classification_continued.Rmd

```diff
@@ -540,17 +638,17 @@ training observations. We can evaluate a classifier by splitting the data
 randomly into a training and test data set, using the training set to build the
```

Comment:
The splitting data section above seems out of place where it is... Should it go above where we split into train and test?

Reply:
Also, from the docs it seems that `initial_split` randomly splits by default (and if you don't want to do that, as in a time series, you would use `initial_time_split`)... 

UPDATE: I now read that we are saying this, but it's not clear to me in the first read through. Can we make it more explicit in this sentence: "The tidymodels function initial_split and vfold_cv functions do this for us."

Reply:
I think the explanation for stratification should be improved, we do not explicitly say how using the strata argument ensures that enough of each class ends up in each of the train, validation, and test partitions.

Reply:
## Summary section:

"Classification algorithms use one or more quantitative variables to predict the value of a third, categorical variable. " shouldn't say "third" because we say "one or more", it should say "another"

Reply:
"To tune the classifier (e.g., select the K in K-nearest neighbours), we maximize accuracy estimates from cross-validation. " should say "We can tune the classifier (e.g., select the K in K-nearest neighbours) by maximizing accuracy estimates from cross-validation. " I suggest this change as we could use some other metric to maximize, we just choose to use accuracy here for simplicity in this book.

Reply:
done